src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -23,181 +28,341 @@ import Screenshots from '../Screenshots'
-  const {
-    name,
-    instances,
-    baseUrl,
-    currentVersion,
-    latestVersion: {
-      content: {
-        author,
-        icons,
-        description = 'No description.',
-        screenshots = [],
-        source_url: sourceUrl,
-        details_url: detailsUrl,
+const AppContent = React.memo(
+  ({ repo, repoVersions, onRequestUpgrade, onClose }) => {
+    const theme = useTheme()
+    const {
+      name,
+      instances,
+      baseUrl,
+      currentVersion,
+      latestVersion: {
+        content: {
+          author,
+          icons,
+          description = 'No description.',
+          screenshots = [],
+          source_url: sourceUrl,
+          details_url: detailsUrl,
+        },
+        version: latestVersion,
       },
-      version: latestVersion,
-    },
-    repoAddress,
-    repoName,
-  } = repo
-  const repoDetails = useRepoDetails(baseUrl, detailsUrl)
-  const canUpgrade = currentVersion.version !== latestVersion
-  const { below, breakpoints } = useViewport()
-  const compact = appBelow(below, breakpoints.medium)
-
-  return (
-    <div
-      css={`
-        padding: ${6 * GU}px ${4 * GU}px ${8 * GU}px;
-      `}
-    >
-      <div
-        css={`
-          display: flex;
-          justify-content: space-between;
-          margin-bottom: ${6 * GU}px;
-          overflow: hidden;
-          flex-direction: ${compact ? 'column' : 'row'};
-          align-items: ${compact ? 'flex-start' : 'flex-end'};
-        `}
-      >
+      repoAddress,
+      repoName,
+    } = repo
+    const repoDetails = useRepoDetails(baseUrl, detailsUrl)
+    const canUpgrade = currentVersion.version !== latestVersion
+    const { below, breakpoints } = useViewport()
+    const compact = appBelow(below, breakpoints.medium)
+
+    return (
+      <React.Fragment>
+        <Bar>
+          <BackButton onClick={onClose} />
+        </Bar>
+        <Split
+          primary={
+            <Box>
+              <div>
+                <div
+                  css={`
+                    display: grid;
+                    grid-template-columns: auto auto 1fr;
+                    grid-gap: ${2 * GU}px;
+                  `}
+                >
+                  <AppIcon app={{ baseUrl, icons }} size={80} />
+                  <div
+                    css={`
+                      display: flex;
+                      flex-direction: column;
+                      align-items: flex-start;
+                      justify-content: center;
+                    `}
+                  >
+                    <h1
+                      css={`
+                        color: ${theme.content};
+                        textStyle('title3');
+                        white-space: nowrap;
+                        margin-bottom: ${1 * GU}px;
+                      `}
+                    >
+                      {name}
+                    </h1>
+                    {author && (
```

Comment:
![image](https://user-images.githubusercontent.com/3072458/63502052-043da780-c4cd-11e9-84ad-f05686000c07.png)

@dizzypaty @owisixseven the font style here (`title3`), looks a bit small imo. I'm just looking for confirmation here that this is how we want it, or if we should make it bigger.

Reply:
Hey @AquiGorka this title must be "Title 3" -> 24px.
https://devbox.dweb.cool/#TextStyles

Reply:
Yeah, that's what I was aiming for but had a typo. It is fixed now, thanks @owisixseven